lib/msal-browser/src/index.ts

```diff
@@ -12,14 +12,21 @@
  * Warning: This set of exports is purely intended to be used by other MSAL libraries, and should be considered potentially unstable. We strongly discourage using them directly, you do so at your own risk.
  * Breaking changes to these APIs will be shipped under a minor version, instead of a major version.
  */
-import * as internals from "./internals";
-export { internals };
+/*
```

Comment:
Are these commented imports removable at this point?

Reply:
yes.  They are.... I'll do that

Reply:
Let's make sure the 1P library is set up to handle this

Reply:
Is there a standard approach to synchronizing changes between the 2 repos?  Or to verify that changes to public will not break 1P?

Reply:
Clone the 1P repo, pull in this branch and then attempt to build

Reply:
Should I add an Azure DevOps pipeline for this?  Or trigger a 1P build?

Reply:
I believe that's something @sameerag is working on this quarter. We've been validating manually up to this point.

Reply:
@shoatman We need to track this. For now, at least, we need to verify compilation before we merge, else 1P will be blocked from release.

Reply:
I misspoke. As this is going in v3, should be good for now.